src/mainboard/sifive/hifive/Makefile.toml

```diff
@@ -43,3 +43,12 @@ args = ["-m", "512m", "-machine", "sifive_u", "-nographic", "-bios", "${TARGET_D
 dependencies = ["build"]
 command = "cargo"
 args = ["objdump", "--", "-d", "${TARGET_DIR}/hifive"]
+
+[tasks.flash]
+dependencies = ["default"]
+script = [
+       "sudo flashrom -V -r hifive_original.rom -p dediprog",
```

Comment:
this doesn't work if we have our own compiled flashrom, like I do.

Reply:
You'll have to add your flashrom to your root PATH or put flashrom in a directory like /usr/local/bin